fe1-web/src/core/components/PoPTouchableOpacity.tsx

```diff
@@ -1,19 +1,19 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import { Pressable, StyleProp, View, ViewStyle } from 'react-native';
-import { TouchableOpacity } from 'react-native-gesture-handler';
+import { StyleProp, TouchableOpacity, View, ViewStyle } from 'react-native';
 
 import { ExtendType } from 'core/types';
-
 /**
  * Wraps a touchable opacity in a pressable component to make karate tests work :)
+ * /!\ no longer wraps due to UI bugs (dayan9265, 06-06-2023)
  */
 const PoPTouchableOpacity = React.forwardRef<View, IPropTypes>(
-  ({ onPress, containerStyle, style, testID, children }: IPropTypes, ref) => {
+  ({ onPress, style, testID, children }: IPropTypes, ref) => {
     return (
-      <Pressable onPress={onPress} style={containerStyle} ref={ref} testID={testID || undefined}>
-        <TouchableOpacity style={style}>{children}</TouchableOpacity>
-      </Pressable>
+      // @ts-ignore - type problem with onPress
```

Comment:
It should be fine to remove this and make the `onPress` required? Or alternatively add the default value `() => {}`
```suggestion
```